python/pyspark/sql/tests/connect/test_connect_plan_only.py

```diff
@@ -70,6 +70,14 @@ def test_filter(self):
         self.assertEqual(plan.root.filter.condition.unresolved_function.parts, [">"])
         self.assertEqual(len(plan.root.filter.condition.unresolved_function.arguments), 2)
 
+    def test_filter_with_string_expr(self):
+        """SPARK-41297: filter supports SQL expression"""
```

Comment:
just dropping a comment but it should be:

```
# SPARK-41297: filter supports SQL expression
```

per https://spark.apache.org/contributing.html